libraries/standard/http/utest/http_utest.c

```diff
@@ -176,6 +303,8 @@ void suiteSetUp()
 /* called at the end of the whole suite */
 int suiteTearDown( int numFailures )
```

Comment:
We should probably just delete these since they are not used. Unity will use the default ones if a symbol doesn't exist.

Reply:
Removed